test/RequestFactoryTestCase.php

```diff
@@ -38,7 +38,7 @@ protected function assertRequest($request, $method, $uri)
         $this->assertSame($uri, (string) $request->getUri());
     }
 
-    public function dataMethods()
+    public static function dataMethods()
```

Comment:
Why are these made static? Is that a PHPUnit 10.x thing?

Reply:
> Is that a PHPUnit 10.x thing?

Yes

Reply:
Dataproviders need to be static in PHPUnit 10